### PR TITLE
prep v0.2.2

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
Signed-off-by: Raffaele Spazzoli <raffaele.spazzoli@gmail.com>

### CODE REVIEW

File: deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+              "node-role.kubernetes.io/worker": ""
+            }
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Networking, Cloud Provider
+    certified: "false"
+    containerImage: quay.io/redhat-cop/keepalived-operator:latest
+    createdAt: 02/04/2020
```
Comment (sabre1041) [inline deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml:25]: `containerImage` and `createdAt` need to be updated

File: deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+  - name: repository
+    url: https://github.com/redhat-cop/keepalived-operator
+  - name: containerImage
+    url: https://quay.io/redhat-cop/keepalived-operator:v0.2.2
+  maintainers:
+  - email: rspazzol@redhat.com
+    name: Raffaele Spazzoli
+  maturity: alpha
+  provider:
+    name: Containers & PaaS CoP
```
Comment (sabre1041) [inline deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml:296]: Please update to "Red Hat Community of Practice"

Comment (raffaelespazzoli) [reply deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml]: done

File: deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml (hunk 1 above)
Comment (raffaelespazzoli) [reply deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml]: done

**Comment by raffaelespazzoli** [comment]
fixes #25

**Pull request merged**